Fixed #32060 -- Added Random database function. (#13473)
django/django: docs/releases/3.2.txt

  large number of pages, this can be helpful for generating a reasonable number
  of page links in a template.

> * Added :class:`~django.db.models.functions.Random` database function.

Reviewer: I've moved this to the "Models" section.

  non-deterministic collations are not supported, set
  ``supports_non_deterministic_collations`` to ``False``.

> * ``DatabaseOperations.random_function_sql()`` has been removed in favor of the

Reviewer: ```suggestion
* ``DatabaseOperations.random_function_sql()`` is removed in favor of the
```
